spec/cli.spec.js

```diff
@@ -50,19 +56,34 @@ describe("cordova cli", function () {
             beforeEach(function () {
             });
 
-            it("will spit out the version with -v", function () {
-                cli(["node", "cordova", "-v"]);
-                expect(console.log.mostRecentCall.args[0]).toMatch(version);
+            it("will spit out the version with -v", function (done) {
+                cli(["node", "cordova", "-v"]).then(function() {
+                    expect(console.log.mostRecentCall.args[0]).toMatch(version);
+                    done();
+                }).fail(function() {
```

Comment:
They have changed because ```telemetry.showPrompt()``` is async, therefore the return of ```cli``` turned into async, and the tests had to be updated to reflect that.